### ISSUE TITLE
Limit getAzureAgentTemplate log spam

### ISSUE THREAD

**Issue by shanep2300** [jenkinsci/azure-vm-agents-plugin#538]
### What feature do you want to see added?

Some times very hard to follow the logs as this just takes up a lot of the output. Currently have it set to FINE, but some FINE logs are needed. Have it output every 5-10 minutes? Seems to be every 20-30s.

**Comment by shanep2300** [comment]
We have around 15 templates. (Could be different for others) but that's around 30+ lines every 20-30s. Only reason I created this is that we're currently debugging an issue and most of the page is taken up by getAzureAgentTemplate. When done, we'll revert from FINE.

It's just the consistency of that output.

**Comment by timja** [comment]
you're welcome to send a PR if it makes it easier for you to debug.
